Cargo.toml

```diff
@@ -33,10 +33,10 @@ protobuf = { version = "2.0", optional = true }
 regex = "1.3"
 reqwest = { version = "0.10", features = ["blocking"], optional = true }
 thiserror = "1.0"
-parking_lot = "0.10.2"
+parking_lot = "^0.11"
```

Comment:
What is the reasoning for not writing `0.11` here as both imply a caret requirement?

> The string "0.1.12" is a semver version requirement. Since this string does not have any operators in it, it is interpreted the same way as if we had specified "^0.1.12", which is called a caret requirement.

https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html

Reply:
They are the same thing, this only makes the "everything 0.11.x" explicit.

Reply:
I am fine either way. Would you mind adjusting all dependency lines in order to stay consistent?

Reply:
I've pushed a commit on top of this to align the semver entries and also alphabetically sort lines, tell me if you like it.